packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -3928,7 +3929,8 @@ describe('src/cy/commands/actions/click', () => {
   })
 })
 
-describe('shadow dom', () => {
+// TODO(webkit): fix+unskip for experimental webkit
```

Comment:
Surprised these tests don't "just work" - I would have thought basic browser APIs would be fairly uniform, since we trigger them with standard client side JS (assuming here, haven't looked at how `cy.shadow` actually works).

Reply:
Actually this passed for me

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/19196536/185279364-abe9d98d-583a-4758-adf0-90fec22b85bf.png">

Reply:
I skipped out the entire click.cy.js because of the large number of failures, I could've been more selective here. This will be unskipped by `experimentalWebKitRelease`.